<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputPanel</class>
 <widget class="QDialog" name="InputPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>299</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>WenQuanYi Micro Hei</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="panelButton_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_9">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_Dot">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_CR">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>81</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Micro Hei</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>CR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_BS">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei Sharp</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退格</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>BS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_ESC">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei Mono</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>ESC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panelButton_Dot_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
   <property name="ButtonValue" stdset="0">
    <string>.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
